src/appleseed/renderer/kernel/rendering/debug/blanktilerenderer.cpp

```diff
@@ -62,6 +62,10 @@ namespace
       : public ITileRenderer
     {
       public:
+        void print_settings() const override
```

Comment:
Minor nitpick: this should go after the `release()` method which is part of an inherited interface, `foundation::IUnknown`.